test/dbTest.test.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose');
+const { MongoClient } = require('mongodb');
+const Item = require('../database/itemModel');
+
+describe('insert', () => {
+  let connection;
+
+  beforeAll(async () => {
+    connection = await MongoClient.connect(process.env.MONGO_URL,
+      { useNewUrlParser: true, useUnifiedTopology: true });
+    await connection.db();
+  });
+
+  afterAll(async () => {
+    await connection.close();
+    await mongoose.connection.close();
+  });
+
+  afterEach(async () => {
+    await Item.deleteMany();
+  });
+
+  it('should insert a doc into collection with right shape', async () => {
+    const item = await new Item({
```

Comment:
No need to await the creation of an object if it doesn't use anything asynchronously